Repositories/News.cs

```diff
@@ -9,7 +9,7 @@ namespace Repositories
     public class NewsRepository
     {
         DatabaseContext Db;
-        News Entity;
+        News[] Entity;
```

Comment:
Since this repo is for news entirely, it's better avoid using Entity or Entities, because it's not very descriptive. use News everywhere instead